rasa/shared/core/training_data/story_writer/yaml_story_writer.py

```diff
@@ -228,7 +229,29 @@ def process_user_utterance(
             entities = []
             for entity in user_utterance.entities:
                 if "value" in entity:
-                    entities.append(OrderedDict([(entity["entity"], entity["value"])]))
+                    if hasattr(user_utterance, "inline_comment_for_entity"):
+                        for predicted in user_utterance.predicted_entities:
+                            if user_utterance.inline_comment_for_entity(
```

Comment:
If you put this into a variable you won't have to call the same method again on line 241